src/main/java/lv/ctco/javaschool/goal/boundary/GoalApi.java

```diff
@@ -152,7 +154,6 @@ public void editGoal(@PathParam("id") Long goalId, GoalFormDto newGoalDto) {
     }
 
     @POST
-    @RolesAllowed({"ADMIN", "USER"})
```

Comment:
Why was this removed? Do you want to allow users without these roles to use these methods?

Reply:
they were moved to the top of the GoalApi because that all the methods are allowed to all users in GoalApi .

Reply:
OK. Did not see the class-level annotation at first, due to GitHub collapsing unchanged code.